### DISCUSSION TITLE
Input zipped fastq or aligned bam file for Isoquant

### DISCUSSION THREAD

**Discussion by Vibha1277** [ablab/IsoQuant#214] [Q&A]
Hi Team,
I have  pacbio .ccs reads for isoseq count and then DEG analysis.
Should input isoseq fastq.gz file go through trimming the primers or adapter removal and or polyA tail removal first begore running for isoseq count?

 Thanks and Regards,
Vibha

**Comment by andrewprzh** [comment]
Dear @Vibha1277 

Removing adapters and primers would be nice, but it's better to keep polyA tails since IsoQuant relates on those when detecting transcript end sites.

Best
Andrey
